client/components/product-card/index.jsx

```diff
@@ -59,14 +60,14 @@ const ProductCard = ( {
 				</div>
 			</div>
 			<div className="product-card__description">
+				{ description && <p>{ description }</p> }
 				{ purchase && (
-					<p>
-						<a href={ managePurchase( purchase.domain, purchase.id ) }>
-							{ translate( 'Manage Subscription' ) }
-						</a>
-					</p>
+					<ProductCardAction
+						href={ managePurchase( purchase.domain, purchase.id ) }
+						label={ translate( 'Manage Subscription' ) }
+						isPrimary={ false }
```

Comment:
Following @tyxla's suggestions in other places:
```suggestion
						primary={ false }
```